nixos/modules/hoss.nix

```diff
@@ -43,9 +50,35 @@
     cudatoolkit
     devenv
     direnv
+    nvidia-container-toolkit
     zenith-nvidia
   ];
 
+  users = {
+    groups = {
+      # huggingface users
+      hf.name = "hf";
+    };
+    users = {
+      chrash = {
+        extraGroups = [
+          "hf"
+        ];
+        packages = with pkgs; [
+          python313Packages.huggingface-hub
+          mistral-rs
+        ];
+      };
+      mistral = {
+        description = "user to restrict the mistralrs-server";
+        group = "hf";
+        isSystemUser = true;
+        homeMode = "774";
+        home = "/mnt/space/mistral";
+      };
```

Comment:
created a user just to run the container. this is probably too much work on this container, and i should just learn how to write derivations for stuff like this.